presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -116,13 +116,18 @@ public Builder addContainer(String name, DockerContainer container)
             checkState(!containers.containsKey(name), "Container with name %s is already registered", name);
             containers.put(name, requireNonNull(container, "container is null"));
 
-            String containerName = "ptl-" + name;
+            String containerName = "ptl-" + this.name + "-" + name;
```

Comment:
This is cool from naming perspective, but this harms usability of environments. 

I will no longer be able to conveniently `docker exec -it ptl-hadoop-master bash`.

OTOTH, I do not yet appreciate the benefits this gives us.

Reply:
what a problem? you don't even have to type it: docker ps + double click container name + middle mouse button

Reply:
I know where i can find it. I used to do so and was happy to stop doing so.
I actually usually avoid typing it, and use `ctrl+R` shell history search to complete this.
But at times i have to type (eg new command or whatever).

Since I am using different envs (eg singlenode vs singlenode-hdp3), fixed name helps in both cases.

Reply:
you can use something like `ptl-${ENV}-container`, switching environment is changing `ENV`

Reply:
I don't understand this. 
For the record, I am starting env like
```
presto-product-tests-launcher/bin/run-launcher env up --environment singlenode-hdp3 --debug
```
but then the ENV var is not defined.

Reply:
ENV=xxxx
...launcher ... - - environment $ENV ...
docker exec pt-$ENV-yyyy

Just change ENV to switch environment. This way history or scripts will work like as they did.